migrations/20201013183754_mod-statuses.sql

```diff
@@ -0,0 +1,10 @@
+-- Add migration script here
+CREATE TABLE statuses (
+    id bigint PRIMARY KEY,
+    status varchar(500)
+);
```

Comment:
Added docs to the `ModStatus` struct, changed types.

Reply:
1. The `StatusId` wrapper should stay, and instead contain `i32`s.  Ids don't have to be publicly exposed to have to be representable internally.
2. Is there any reason for id being `integer` rather than `serial`? `serial` would make it autoincrement so that you don't have to specify the id of every status you insert.
3. Status is still limited to 500 characters.  Is there any reason for this?

Also, adding more migrations for this one PR makes it much less readable.  At the end, we should be able to merge them all into one migration, since none of this has been run on the main database.